Drop-In/src/main/java/com/braintreepayments/api/dropin/view/EditCardView.java

```diff
@@ -114,6 +114,11 @@ public void setMaskCvv(boolean mask) {
         mCardForm.maskCvv(mask);
     }
 
+    public boolean isEditCardError(ErrorWithResponse errors) {
+        return errors.errorFor("unionPayEnrollment") != null ||
```

Comment:
why do we need to check for UnionPay specifically here?

Reply:
In the case of union pay, there is no `creditCard` key and all errors are nested under the `unionPayEnrollment` key.  This checks both to determine if there are errors in either case.